style.css

```diff
@@ -367,3 +367,93 @@ h3 {
   transform: rotate(45deg);
   -webkit-transform: rotate(45deg);
 }
+
+.contact-form {
+  background-color: #fff;
+}
+
+.Contact_form {
+  background-image: url(./images/contact_form.png);
```

Comment:
- [ ] Kindly make sure in your `images` folder you have this `contact_form.png` image. At this moment, our website can't find this particular image. Kindly make sure to add it as well.

    - ### Your design
    ![image](https://user-images.githubusercontent.com/87197412/193867121-d8ff7a37-da67-4584-a2ac-00939415fe0f.png)

    - ### Figma design

   ![image](https://user-images.githubusercontent.com/87197412/193867443-62083916-6d94-4285-a139-ac50f45a0ee5.png)

Reply:
Added